client/api_key_project_assignment.go

```diff
@@ -0,0 +1,33 @@
+package client
+
+type ApiKeyRole string
+
+const (
+	ApiKeyRoleDeployer ApiKeyRole = "Deployer"
+	ApiKeyRoleViewer   ApiKeyRole = "Viewer"
+	ApiKeyRolePlanner  ApiKeyRole = "Planner"
+	ApiKeyRoleAdmin    ApiKeyRole = "Admin"
+)
+
+type AssignAPIKeyToProjectPayload struct {
+	UserId string     `json:"userId"`
+	Role   ApiKeyRole `json:"role"`
+}
+
+type ApiKeyProjectAssignment struct {
+	Id string `json:"id"`
+}
+
+func (client *ApiClient) AssignApiKeyToProject(projectId string, payload *AssignAPIKeyToProjectPayload) (*ApiKeyProjectAssignment, error) {
```

Comment:
These endpoints aren't specific to api keys, they are for assigning users to projects, and a "non admin api key" is just a user at the end of it. 

WDYT about renaming this file/methods to reflect that?

Reply:
I think it's a good idea. I'll make some changes.